src/LHScriptX/CommandSignature.h

```diff
@@ -37,12 +38,10 @@ class ScriptCommandParameter
 	    : _type(type)
 	{
 	}
-	ScriptCommandParameter(const std::string value)
+	ScriptCommandParameter(const std::string_view& value)
```

Comment:
Pass string_view by value and without const

Reply:
Reverted to string.